README.md

```diff
@@ -23,6 +23,13 @@ After the compilation you'll receive libmarine.so which you can then load using
 ## Usage
 The usage is described in C, however it can be applied to any language that will load libmarine.so and call its functions.
 
+0. Possible errors for initializing Marine (calling init_marine()):
+
+    | Value         | Condition     |
+    | ------------- |:-------------:|
+    | -2    | Marine was already initialized               |
```

Comment:
`init_marine` has more possible errors (from the internal call to `_init_marine`). If you decide that the difference between them isn't interesting, merge all of the error codes to a single "internal error", but do document it :)